composer.json

```diff
@@ -90,15 +90,16 @@
         "composer/installers": "1.12.0",
         "cweagans/composer-patches": "1.7.3",
         "danskernesdigitalebibliotek/cms-api": "*",
-        "danskernesdigitalebibliotek/dpl-design-system": "2024.28.0",
-        "danskernesdigitalebibliotek/dpl-react": "2024.28.0",
+        "danskernesdigitalebibliotek/dpl-design-system": "^2024.28",
+        "danskernesdigitalebibliotek/dpl-react": "^2024.28",
```

Comment:
I wonder what is up with this change. It does not seem to be related to the change at hand.

Reply:
I had merge conflicts in composer.lock, which I solve by merging develop into my branch, accept incoming changes. After this I run a raycast-script which looks like this:

`task dev:cli -- composer remove danskernesdigitalebibliotek/dpl-react danskernesdigitalebibliotek/dpl-design-system \
&& task dev:cli -- composer require danskernesdigitalebibliotek/dpl-react danskernesdigitalebibliotek/dpl-design-system`